### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jkleiser**
Several minor corrections. Added a few commas to make reading easier. Please review carefully, as my understanding of the matter is still not the best.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,22 +203,21 @@ could affect the position of the inserted elements.
...
-The hook is only triggered when and element is to be removed from its parent –
-not if it is the child of an element that is removed. For that see the destroy
+The hook is only triggered when an element is to be removed from its parent –
+not if it is the child of an element that is removed. For that, see the destroy
 hook.
 
 #### The `destroy` hook
 
-This hook is invoked whenever an element removed from the DOM or the child
+This hook is invoked whenever an element removed from the DOM or from the child
```
Comment (paldepind) [inline README.md]: I'm not sure if either of these are understandable. The `destroy` hook is invoked on a virtual node when its element is being removed from the DOM or when it is the child of an element that is being removed from the DOM.

**Comment by paldepind** [comment]
Very great changes! Thank you.

I have left a single comment.

File: README.md (hunk 1 above)
Comment (jkleiser) [inline README.md]: Could you yourself just put in the words that you think is most correct, or do you prefer that I do the rephrasing according to your suggestion?

**Comment by jkleiser** [comment]
Could you yourself just put in the words that you think is most correct, or do you prefer that I do the rephrasing according to your suggestion?

File: README.md [hunk 2]
```diff
@@ -280,22 +279,22 @@ h('a', {props: {href: '/foo'}}, 'Go to Foo');
...
 ```
 
 Attributes are added and updated using `setAttribute`. In case of an attribute
-that has been previously added/set is no longer present in the `attrs` object,
+that has been previously added/set and is no longer present in the `attrs` object,
 it is removed from the DOM element's attribute list using `removeAttribute`.
 
-In the case of boolean attributes (.e.g. `disabled`, `hidden`, `selected` ...).
-The meaning doesn't depend on the attribute value (`true` or `false`) but depends
+In the case of boolean attributes (.e.g. `disabled`, `hidden`, `selected` ...),
```
Comment (jkleiser) [inline README.md]: Should be no "." before "e.g.".

**Comment by paldepind** [comment]
I've merged the PR and fixed the two issues discussed above.

Thank you a lot for these improvements to the readme :+1:

**Pull request merged**